webclient/components/pt-info/single/1time-Mrow-mField/diagnosis/view-layer/table-ct.vue

```diff
@@ -0,0 +1,51 @@
+<template>
```

Comment:
This page need to be upgrade with new architecture.
Ref: webclient/components/pt-info/single/1time-Mrow-1Field/reminder/view-layer/grid-design2-ct.vue